src/app/chat/UserCard.vue

```diff
@@ -383,7 +383,7 @@ function openNativeCard(ev: MouseEvent): void {
 }
 
 function getProfileURL(): string {
-	return window.location.origin + "/" + props.target.username;
+	return "https://www.twitch.tv/" + props.target.username;
```

Comment:
This shouldn't be hardcoded, as there is no guarantee the current origin is `twitch.tv`. Use location data instead.

Reply:
> This shouldn't be hardcoded, as there is no guarantee the current origin is `twitch.tv`. Use location data instead.

I propose this solution to the problem

```ts
function getLocationOrigin(): string {
  if (location.hostname.includes("dashboard")) {
    return "https://www.twitch.tv/";
  }

  return location.origin;
}
```